src/app/components/listing/details/criteria/edit.component.js

```diff
@@ -74,6 +77,19 @@ export const CertificationCriteriaEditComponent = {
             this.close({$value: this.cert});
         }
 
+        svapOnChange (action) {
+            switch (action.action) {
+            case 'Remove':
+                this.cert.svaps = this.cert.svaps
+                    .filter(svap=> !(svap.svapId === action.item.item.svapId));
```

Comment:
```suggestion
                    .filter(svap => svap.svapId !== action.item.item.svapId);
```
The "test data" filter has the "not (this and this and this)" construct because there are multiple values to check. Test standard, which is similar in that it only has the standard, just does the id !== id check. Request the simpler check